backend/src/models/__test__/user.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { User } from '../user'
+import { TestDatabase } from './TestDatabase'
+import { clone } from '../../utils/clone'
+
+describe('Testing user component', () => {
+  const database = new TestDatabase()
+  const user = clone(new User())
+
+  user.email = 'x@email.com'
+  user.firstname = 'Vic'
+  user.lastname = 'Sidious'
+  user.password = 'pieTillIDie'
+
+  beforeEach(async () => {
+    await database.init()
+  })
+
+  afterEach(async () => {
+    await database.kill()
+  })
```

Comment:
Database/server connection should be cleaned up, don't forget about this when testing.

Reply:
Well noted